### PR TITLE
2022-11-17- project-update-draft-interface.md

### PR DESCRIPTION
**Pull request by Prakash-sa**
This is the update of Prakash-sa's final project and the draft of the Stock market user interface.

### CODE REVIEW

**Review by hannahmoutran — COMMENTED** [review]
Hi Prakash, I think for the post title, you're missing this at the beginning: inf380p.github.io...also, I'm not sure, but the 'Create' before the date might mess things up.  Also the spaces.  In my notes, I have it written like this for reference: inf380p.github.io/_posts/hannahmoutran/2022-09-29-first-focused-reflection.md

File: _posts/Prakash-sa/Create 2022-11-17- Project Update: Draft Interface.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# Final Project Interface Start
+
+For the final project, as stated in the inital project proposal, I am planning to do the data processing. I am going to do the web scraping of https://www.tradingview.com/. 
+I will do the Data Cleaning, Visualization and Storing with the python programming.
+
+I will clean and visualize the Stock market data from the different country. It will include the feature of the performance, rise and fall, Large cap, Small cap, Top gainers, Most active, etc.
+
+- For the data, I am using the HTML format from the website.
+
+I feel the following can be a good plan to create the project:
```
Comment (hannahmoutran) [inline _posts/Prakash-sa/Create 2022-11-17- Project Update: Draft Interface.md:16]: Hi Prakash!  Instead of "can be" I would just say "is"

Comment (hannahmoutran) [reply _posts/Prakash-sa/Create 2022-11-17- Project Update: Draft Interface.md:16]: It's so cool that you're scraping the data yourself!

Comment (Prakash-sa) [reply _posts/Prakash-sa/Create 2022-11-17- Project Update: Draft Interface.md]: Thank's Hannah, I have done the changes.

**Review by Rowan-Loft — APPROVED** [review]
Thanks for fixing the title for proper formatting. Should save us some issues with how it appears on the class.

**Pull request merged**